deeplabcut/pose_estimation_tensorflow/core/train_multianimal.py

```diff
@@ -63,7 +65,8 @@ def train(
     cfg["pseudo_threshold"] = pseudo_threshold
     cfg["video_path"] = video_path
     cfg["traintime_resize"] = traintime_resize
-
+    cfg["topview"] = topview
```

Comment:
can you explain why this is needed?

Reply:
To add gradient masking in video adaptation. If the user choose high threshold for video adaptation, then many keypoints are dropped thus masking is needed. But "masking for video adaptation" is only applicable for topview. During inference time,  masking for  side-view quadruped models will make the model confuse occluded keypoints and visible but filtered keypoints .

Reply:
lets minimally change to ["superanimal"] then topview is okay; then we are forward thinking for new models. Agree @yeshaokai then we can merge actually